hermes-mock/src/test/groovy/pl/allegro/tech/hermes/mock/TestMessage.groovy

```diff
@@ -0,0 +1,51 @@
+package pl.allegro.tech.hermes.mock
+
+import com.fasterxml.jackson.core.JsonProcessingException
+import com.fasterxml.jackson.databind.ObjectMapper
+
+class TestMessage {
+    private static final ObjectMapper objectMapper = new ObjectMapper();
+
+    private String key
+    private String value
+
+    String getKey() {
```

Comment:
In Groovy you don't have to write getters and setters. All public (if you don't write modifier then field is public) fields have getters and setters. Check [@Canonical annotation](http://docs.groovy-lang.org/2.4.7/html/gapi/groovy/transform/Canonical.html)